_episodes/08-putting-it-all-together.md

```diff
@@ -146,25 +146,27 @@ styles and the source codes that create them.
 
 ### `plt` pyplot versus object-based matplotlib
 
-Matplotlib integrates nicely with the Numpy package and can use Numpy arrays
+Matplotlib integrates nicely with the NumPy package and can use NumPy arrays
 as input of the available plot functions. Consider the following example data,
-created with Numpy:
+created with NumPy by drawing samples from a normal distribution:
```

Comment:
```suggestion
created with NumPy by drawing 1000 samples from a normal distribution with a mean of 0 and a sigma of 0.01:
```

Reply:
I think it's better to spell out `standard deviation` instead of using just `sigma`